src/components/prescriptions/ImportGroupComponent.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+    <section class="prescription-container">
+        <div class="prescriptionContainer">
+            <h3 class="prescriptionContainer__header">
+                Zaimportowane recepty
+                <span class="numOfPres">{{
+                    prescriptionImportList?.length
+                }}</span>
+            </h3>
+            <ul
+                v-if="prescriptionImportList?.length"
+                class="prescriptionContainer__list">
+                <prescription-import-component
+                    v-for="presc in prescriptionImportList"
+                    :key="presc.id"
+                    :testUrl="presc.testUrl"
+                    :name="presc.doctorDataImport"
+                    :date="presc.dateImport"
+                    :typeofdoc="presc.typeOfDoc"
+                    :medicine="presc.medicineNameImport"
+                    :frequency="presc.frequencyMedicineImport" />
+            </ul>
+            <div
+                v-else
+                class="fallback">
+                <fallback-info-component :information="importInfo" />
+            </div>
+        </div>
+    </section>
+</template>
+
+<script setup>
+    import { storeToRefs } from 'pinia';
+    import useUserStore from '../../stores/userStore';
+    import { onMounted, computed, ref } from 'vue';
+    import PrescriptionImportComponent from './PrescriptionImportComponent..vue';
+    import FallbackInfoComponent from '../FallbackInfoComponent.vue';
+
+    const userStore = useUserStore();
+
+    onMounted(async () => {
```

Comment:
Are u sure is it good place to fetch data from backend? What about prescription view?

Reply:
I saw that there you fetch also data? I think it's redundant request.